latch_cli/services/register/register.py

```diff
@@ -228,6 +229,29 @@ def _build_and_serialize(
         exit_status = ctx.dkr_client.wait(container_id)
         if exit_status["StatusCode"] != 0:
             click.secho("\nWorkflow failed to serialize", fg="red", bold=True)
+            if "TypeTransformerFailedError" in "".join(serialize_logs):
+                file_to_update = (
+                    "latch_metadata/parameters.py"
+                    if ctx.workflow_type
+                    in (WorkflowType.nextflow, WorkflowType.snakemake)
+                    else "workflow function"
+                )
+                click.secho(
+                    dedent(f"""
+                    This is likely caused by a type mismatch between the type of one of
+                    your input parameters and the type of the default value for that
+                    parameter.
```

Comment:
this is great - we should also put this in lytekit too so that if this happens at runtime people aren't confused